Default JMH run fails (dummy has no test configuration?)

On a clean clone (commit 1c13d77) the following fails (with JDK 10 but I guess it is not JDK-specific):

```shell
$./tools/sbt/bin/sbt assembly && ./tools/sbt/bin/sbt  renaissanceJmh/jmh:assembly && ./tools/ci/bench-jmh.sh
```

The error shown is

```
Suppressed: java.util.NoSuchElementException: benchmark 'dummy-empty' has no such configuration: test
```

Relevant trace is the following:

```
....

# Run progress: 42,31% complete, ETA 00:02:27
# Fork: 1 of 1
Iteration   1: <failure>

java.util.NoSuchElementException: benchmark 'dummy-empty' has no such configuration: test
	at org.renaissance.core.BenchmarkDescriptor.getConfiguration(BenchmarkDescriptor.java:143)
	at org.renaissance.core.BenchmarkSuite.createBenchmarkContext(BenchmarkSuite.java:113)
	at org.renaissance.jmh.JmhRenaissanceBenchmark.<init>(JmhRenaissanceBenchmark.java:84)
	at org.renaissance.dummy.JmhDummyEmpty.<init>(JmhDummyEmpty.java:14)
	at org.renaissance.dummy.generated.JmhDummyEmpty_jmhType_B1.<init>(JmhDummyEmpty_jmhType_B1.java:3)
	at org.renaissance.dummy.generated.JmhDummyEmpty_jmhType_B2.<init>(JmhDummyEmpty_jmhType_B2.java:3)
	at org.renaissance.dummy.generated.JmhDummyEmpty_jmhType_B3.<init>(JmhDummyEmpty_jmhType_B3.java:2)
	at org.renaissance.dummy.generated.JmhDummyEmpty_jmhType.<init>(JmhDummyEmpty_jmhType.java:2)
	at org.renaissance.dummy.generated.JmhDummyEmpty_runOperation_jmhTest._jmh_tryInit_f_jmhdummyempty0_G(JmhDummyEmpty_runOperation_jmhTest.java:589)
	at org.renaissance.dummy.generated.JmhDummyEmpty_runOperation_jmhTest.runOperation_SingleShotTime(JmhDummyEmpty_runOperation_jmhTest.java:477)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
	at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
	at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
	at java.base/java.lang.reflect.Method.invoke(Method.java:564)
	at org.openjdk.jmh.runner.BenchmarkHandler$BenchmarkTask.call(BenchmarkHandler.java:453)
	at org.openjdk.jmh.runner.BenchmarkHandler$BenchmarkTask.call(BenchmarkHandler.java:437)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:514)
	at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
	at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
	at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
	at java.base/java.lang.Thread.run(Thread.java:844)


Benchmark had encountered error, and fail on error was requested
ERROR: org.openjdk.jmh.runner.RunnerException: Benchmark caught the exception
	at org.openjdk.jmh.runner.Runner.runBenchmarks(Runner.java:578)
	at org.openjdk.jmh.runner.Runner.internalRun(Runner.java:320)
	at org.openjdk.jmh.runner.Runner.run(Runner.java:209)
	at org.openjdk.jmh.Main.main(Main.java:71)
Caused by: org.openjdk.jmh.runner.BenchmarkException: Benchmark error during the run
	at org.openjdk.jmh.runner.BenchmarkHandler.runIteration(BenchmarkHandler.java:428)
	at org.openjdk.jmh.runner.BaseRunner.runBenchmark(BaseRunner.java:282)
	at org.openjdk.jmh.runner.BaseRunner.runBenchmark(BaseRunner.java:234)
	at org.openjdk.jmh.runner.BaseRunner.doSingle(BaseRunner.java:139)
	at org.openjdk.jmh.runner.BaseRunner.runBenchmarksForked(BaseRunner.java:76)
	at org.openjdk.jmh.runner.ForkedRunner.run(ForkedRunner.java:72)
	at org.openjdk.jmh.runner.ForkedMain.main(ForkedMain.java:84)
	Suppressed: java.util.NoSuchElementException: benchmark 'dummy-empty' has no such configuration: test
		at org.renaissance.core.BenchmarkDescriptor.getConfiguration(BenchmarkDescriptor.java:143)
		at org.renaissance.core.BenchmarkSuite.createBenchmarkContext(BenchmarkSuite.java:113)
		at org.renaissance.jmh.JmhRenaissanceBenchmark.<init>(JmhRenaissanceBenchmark.java:84)
		at org.renaissance.dummy.JmhDummyEmpty.<init>(JmhDummyEmpty.java:14)
		at org.renaissance.dummy.generated.JmhDummyEmpty_jmhType_B1.<init>(JmhDummyEmpty_jmhType_B1.java:3)
		at org.renaissance.dummy.generated.JmhDummyEmpty_jmhType_B2.<init>(JmhDummyEmpty_jmhType_B2.java:3)
		at org.renaissance.dummy.generated.JmhDummyEmpty_jmhType_B3.<init>(JmhDummyEmpty_jmhType_B3.java:2)
		at org.renaissance.dummy.generated.JmhDummyEmpty_jmhType.<init>(JmhDummyEmpty_jmhType.java:2)
		at org.renaissance.dummy.generated.JmhDummyEmpty_runOperation_jmhTest._jmh_tryInit_f_jmhdummyempty0_G(JmhDummyEmpty_runOperation_jmhTest.java:589)
		at org.renaissance.dummy.generated.JmhDummyEmpty_runOperation_jmhTest.runOperation_SingleShotTime(JmhDummyEmpty_runOperation_jmhTest.java:477)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke0(Native Method)
		at java.base/jdk.internal.reflect.NativeMethodAccessorImpl.invoke(NativeMethodAccessorImpl.java:62)
		at java.base/jdk.internal.reflect.DelegatingMethodAccessorImpl.invoke(DelegatingMethodAccessorImpl.java:43)
		at java.base/java.lang.reflect.Method.invoke(Method.java:564)
		at org.openjdk.jmh.runner.BenchmarkHandler$BenchmarkTask.call(BenchmarkHandler.java:453)
		at org.openjdk.jmh.runner.BenchmarkHandler$BenchmarkTask.call(BenchmarkHandler.java:437)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at java.base/java.util.concurrent.Executors$RunnableAdapter.call(Executors.java:514)
		at java.base/java.util.concurrent.FutureTask.run(FutureTask.java:264)
		at java.base/java.util.concurrent.ThreadPoolExecutor.runWorker(ThreadPoolExecutor.java:1135)
		at java.base/java.util.concurrent.ThreadPoolExecutor$Worker.run(ThreadPoolExecutor.java:635)
		at java.base/java.lang.Thread.run(Thread.java:844)
```

@lbulej Do you know what could be causing this?

--- a/benchmarks/dummy/src/main/java/org/renaissance/dummy/DummyEmpty.java
+++ b/benchmarks/dummy/src/main/java/org/renaissance/dummy/DummyEmpty.java
@@ -12,6 +12,7 @@ import static org.renaissance.Benchmark.*;
 @Group("dummy")
 @Summary("A dummy benchmark which only serves to test the harness.")
 @Licenses(License.MIT)
+@Configuration(name = "test")
 public final class DummyEmpty implements Benchmark {
   @Override
   public BenchmarkResult run(BenchmarkContext c) {

